spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/KerasUtils.scala

```diff
@@ -46,15 +47,28 @@ object KerasUtils {
     }
   }
 
-  private[keras] def getActivation[T : ClassTag] (activation: String)
+  private[bigdl] def getKerasActivation[T : ClassTag] (activation: String)
+    (implicit ev: TensorNumeric[T]): KerasLayer[Tensor[T], Tensor[T], T] = {
+    if (activation == null) { return null }
+    if (activation.toLowerCase() == "softmax") {
```

Comment:
what's the difference between Keras softmax and Torch softmax?

Reply:
@jason-dai Torch Softmax is different from Keras Softmax for __3D__ input. They will have the same result for 2D input. https://github.com/intel-analytics/BigDL/issues/1934 
For 3D, currently to be compatible, we first do a transpose before applying softmax.